services/worker/src/worker/job_runners/split/descriptive_statistics.py

```diff
@@ -527,19 +526,24 @@ def compute_descriptive_statistics_response(
             "no features found in 'dataset_info'. "
         )
 
-    split_parquets_size = sum(parquet_file["size"] for parquet_file in split_parquet_files)
-    if split_parquets_size > max_parquet_size_bytes:
-        raise SplitWithTooBigParquetError(
-            f"Statistics computation is limited to split parquets under {max_parquet_size_bytes} bytes. "
-            f"Current size of sum of split parquets is {split_parquets_size} bytes."
-        )
+    num_parquet_files_to_index = 0
+    total_bytes = 0
+    split_directory = split_parquet_files[0]["url"].rsplit("/", 2)[1]
+    partial_parquet_export = parquet_export_is_partial(split_parquet_files[0]["url"])
+    for parquet_file_id, parquet_file in enumerate(split_parquet_files):
+        num_parquet_files_to_index += 1
+        parquet_file_size_bytes = parquet_file["size"]
```

Comment:
btw why in `duckdb-index` worker the size is taken from `config-parquet-metadata` step and read again with `pyarrow.parquet` @lhoestq ? 
i mean this: https://github.com/huggingface/datasets-server/blob/348567390f4dfbcf5d93aec462f07c46d09cd9ad/services/worker/src/worker/job_runners/split/duckdb_index.py#L128

Reply:
to be sure not to forget this, maybe open an issue for duckdb-index?

Reply:
this is actually related to the idea that i misunderstood - that we take size of raw data, while `size` key here corresponds to parquet file size itself. 

would be nice if we stored raw data size too though, to simplify this code but okay :D